production/grafana-agent-mixin/alerts.libsonnet

```diff
@@ -235,6 +235,151 @@ local _config = config._config;
             },
           },
         ],
+      },
+      {
+        name: 'GrafanaAgentManagement',
+        rules: [
+          {
+            alert: 'AgentManagementBadAPIRequests',
+            expr: |||
+              100 * sum(rate(agent_remote_config_fetches_total{status_code!~"2.."}[5m])) by (%(group_by_cluster)s)
```

Comment:
I think it is probably safer to use 10m, I could see agents having multi-minute poll intervals to reduce request rates